pachyderm/templates/pachd/rbac/role.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if and .Values.rbac.create .Values.rbac.localRoles -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Can this be combined with the `clusterrole` definition? ie. does anything change other than the `kind`?

Reply:
It could be, except I think the normal practice is to name each template after the kind of object being created, so have a `ClusterRole` inside a file named `role.yaml` would be _slightly_ odd.

Reply:
hrm, good point. I think perhaps we should change the name though to be a bit more clear. Perhaps `namespacedRole` or `namespaceLocalRole`?

Reply:
Here's an example from nginx stable (not canonical, but perhaps another option) https://github.com/helm/charts/pull/20984/files see `scope`

Reply:
Since k8s itself defines Roles (which are namespaced) and ClusterRoles (which are cluster-wide and not namespaced), maybe the problem is with the flag `--local-roles`: maybe it should actually be `--cluster-roles`, and it should default to true?

Reply:
The remaining question is: should I go with the `pachctl deploy` name `rbac.localRoles = true` here, or go with the (probably more idiomatic) `rbac.clusterRoles = false`?

Reply:
We determined in an offline conversation that `clusterRBAC` is a decent choice.